<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>designer</class>
 <widget class="QWidget" name="designer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>277</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>مدیریت آزمون ها</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>نرم افزار من !!!!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>عنوان آزمون:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="cmbTest"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnEdit">
         <property name="text">
          <string>مشاهده / ویرایش</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDelete">
         <property name="text">
          <string>حذف آزمون</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNew">
         <property name="text">
          <string>آزمون جدید</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCreateUser">
         <property name="text">
          <string>تعریف کاربر</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExit">
         <property name="text">
          <string>خروج</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
